.github/workflows/docsite-publish.yml

```diff
@@ -0,0 +1,55 @@
+# Workflow name
+name: 'Docsite publish'
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    branches:
+      - main
```

Comment:
Oops, this should be `master`, right? Griffel uses `main`...

Reply:
Great catch @layershifter . Fixing now.